cmake/helpers/GdalDriverHelper.cmake

```diff
@@ -299,9 +299,9 @@ function(is_plugin _result _target)
     endif ()
 endfunction()
 
-function(gdal_target_interfaces _TARGET)
+function(gdal_target_interfaces _TARGET _seen_libs)
```

Comment:
Some comment explaining the function arguments would be welcome.
And shouldn't https://github.com/OSGeo/gdal/blob/8fddbf591fa973b5875b8d29f433ce59036e9d53/cmake/helpers/GdalDriverHelper.cmake#L364 be modified to provide the _seen_libs argument ?
I actually do see it in the Conan-center-index pull request: https://github.com/conan-io/conan-center-index/pull/19298/files#diff-46ebe8f12bf2455c0cc64c21467d6d8991953aa4696bc7d425268239694d8ff5R16

Reply:
Indeed, I accidentally moved it to a wrong patch when cleaning things up for this PR. I fixed it now both here and there.

Reply:
Added a comment documenting the function as well now.